src/components/Firestore/Collection.tsx

```diff
@@ -192,17 +194,26 @@ export const CollectionPresentation: React.FC<CollectionPresentationProps> = ({
           ></ListItem>
         </List>
 
-        <List dense className="Firestore-Document-List" tag="div">
+        <List
+          dense
+          className="Firestore-Document-List"
+          tag="div"
+          twoLine={
+            collectionFilter && isSortableCollectionFilter(collectionFilter)
+          }
+        >
           {docs.map(doc => (
-            <ListItem
+            <DocumentListItem
               key={doc.ref.id}
-              className="Firestore-List-Item"
-              tag={NavLink}
-              to={`${url}/${doc.ref.id}`}
-              activeClassName="mdc-list-item--activated"
-            >
-              {doc.ref.id}
-            </ListItem>
+              url={url}
+              docId={doc.ref.id}
+              queryFieldValue={
+                (collectionFilter &&
+                  isSortableCollectionFilter(collectionFilter) &&
+                  get(doc.data(), collectionFilter.field)) ||
```

Comment:
What if the field value is a falsy value like `false` or `0` or `''` itself? Let's do `condition ? value : undefined` and explicitly test for `undefined` in the component.